hadoop-hdfs-project/hadoop-hdfs-rbf/src/site/markdown/HDFSRouterFederation.md

```diff
@@ -288,6 +288,7 @@ For deciding where to create a new file/folder it uses the order parameter, it c
 * RANDOM: Random subcluster. This is usually useful for balancing the load across. Folders are created in all subclusters.
 * HASH_ALL: Follow consistent hashing at all the levels. This approach tries to balance the reads and writes evenly across subclusters. Folders are created in all subclusters.
 * SPACE: Try to write data in the subcluster with the most available space. Folders are created in all subclusters.
+* LEADER_FOLLOWER: Try to write data in the leader sub-cluster as many as possible, if failed, try follower sub-clusters then. Folders are created in all sub-clusters.
```

Comment:
`Try to write data in the leader subcluster as much as possible, if failed, try follower subclusters. Folders are created in all subclusters.`

Reply:
Thanks, updated.